website/docs/flags.md

```diff
@@ -36,7 +36,8 @@ flags { "flag_list" }
 | WPF                   | Mark the project as using Windows Presentation Framework, rather than WinForms. |
 | DebugEnvsDontMerge    | Needs documentation                                                        |
 | DebugEnvsInherit      | Needs documentation                                                        |
-| Thumb                 | Needs documentation                                                        |
+| Thumb                 | Android Only. Generate code that executes for thumb microarchitecture. This is applicable for arm architecture only. |
+| DataLevelLinking      | Android Only. Enables linker optimizations to remove unused data by emitting each data item in a separate section. |
```

Comment:
Seems it is just docu update,
but notice that flags are deprecating for dedicated api.

Reply:
Yeah I get that, there was a stray option in the original vsandroid module and I decided to bring it over. I get that it needs a specific thing but that's probably for another PR

Reply:
FWIW, I think `Thumb` already has a dedicated API but I'm not sure if the flag is deprecated.

Reply:
If thumb has an api already I can try to hook it up too

Reply:
`thumbmode`, uncertain why it doesn't deprecate the flag though both come from the Android module.

Reply:
I addressed this already, it was already using the correct api